src/main/java/org/ice4j/util/RateStatistics.java

```diff
@@ -102,25 +102,45 @@ private void eraseOld(long nowMs)
         oldestTime = newOldestTime;
     }
 
+    public long getRate()
```

Comment:
I'm not sure why we don't have any JavaDoc for (at least) the public methods of this class, but I don't see a reason why we should continue this practice.